laws/src/main/scala/cats/laws/discipline/Eq.scala

```diff
@@ -16,4 +16,11 @@ object eq {
       samples.forall(s => B.eqv(f(s), g(s)) )
     }
   }
+
+  implicit def tuple2Eq[A, B](implicit A: Eq[A], B: Eq[B]): Eq[(A, B)] =
+    new Eq[(A, B)] {
+      def eqv(x: (A, B), y: (A, B)): Boolean =
+        A.eqv(x._1, y._1) && B.eqv(x._2, y._2)
+    }
+
```

Comment:
I agree that it's smart to define the instances we need here, and then remove them when algebra adds them.